tests/components/wallbox/test_init.py

```diff
@@ -58,7 +62,14 @@ async def test_wallbox_refresh_failed_invalid_auth(
         )
         mock_request.put(
             "https://api.wall-box.com/v2/charger/12345",
-            json=json.loads(json.dumps({CHARGER_MAX_CHARGING_CURRENT_KEY: 20})),
+            json=json.loads(
+                json.dumps(
+                    {
+                        CHARGER_MAX_CHARGING_CURRENT_KEY: 20,
+                        CHARGER_ENERGY_PRICE_KEY: 0.3,
+                    }
```

Comment:
I believe that you should mock another http address for price changes, bases on the wallbox pypi package it should be  f"{self.baseUrl}chargers/config/{chargerId}" (https://api.wall-box.com/chargers/config/12345). However, that doesn't seem to fix the problem for me.

Reply:
Tried updating the energy cost in a browser and I am seeing this in my network tracing.
<img width="629" alt="image" src="https://github.com/home-assistant/core/assets/39677/60474bf3-3acf-4fa2-acb4-2f6bef5c7a49">

<img width="629" alt="image" src="https://github.com/home-assistant/core/assets/39677/0eb6e016-74dc-47a7-ad8a-d993d6ef5eba">

Reply:
Also, looks like it's a requests.PUT in the PR at the moment whereas the browser performs a requests.POST.

Reply:
Should be fixed now.